scripts/get_clean_obs_function.R

```diff
@@ -160,6 +169,26 @@ get_clean_obs = function(genus = NULL, species = NULL) {
   occ_df
 }
 
+#' Test whether \code{longitude} is within the bounds of \code{lonbounds}
```

Comment:
@jcoliver What are you doing with the syntax here? Things are showing up in interesting ways in when I open it in RStudio, but I'm not familiar with this type of commenting?

Reply:
roxygen!  [http://r-pkgs.had.co.nz/man.html](http://r-pkgs.had.co.nz/man.html). It's a nice way to document functions that will make generation of documentation easier down the line.